background-concealment-demo-py/BackgroundConcealmentDemo.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    g_bgr      = cv.gapi.blur(cv.gapi.resize(g_in, in_size), blur_sz)
</pre><p>I think, it's better to name it <code>blurred</code> or smth like that</p><p>It's not a background yet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TolyaTalamanov/g-api-collection/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>